lib/rules/web/admin_console/4.1/route.xyaml

```diff
@@ -0,0 +1,52 @@
+create_route:
+  action: set_route_name
+  action: set_service_name
+  action: set_target_port
+  action:
+    if_param:
+      secure_route: "true"
+    ref: set_secure_route
+  action: click_create_button
+
+set_route_name:
+  element:
+    selector:
+      xpath: //input[@id='name']
+    op: send_keys <route_name>
+    type: input
+set_service_name:
```

Comment:
This can be replaced with existing rules:
```
set_service_name:
  params:
    dropdown_field: Service
    dropdown_item: <service_name>
  action: choose_dropdown_item
```

Reply:
what about this comment?